i18n/ja/README.md

```diff
@@ -51,6 +52,9 @@ Kubernetes Dashboard の和訳を実施するにあたって、下記の配慮
 * 英単語と日本語の間には半角スペースを入れます。
   + (例) ○ `これは English word です。`
   + (例) × `これはEnglish wordです。`
+  + ただし、文頭、句読点の直前直後、では、半角スペースを挿入しません。
+  + (例) ◯ `これは English、Chinese、French です。`
```

Comment:
`◯` is different from `○`. `○` is used at other place In this doc.
`◯`  : L56
`○` : L50, L53, L64

Reply:
thx. done.